ios-competency-test/ios-competency-test/ViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ViewController.swift
+//  ios-competency-test
+//
+//  Created by Sam Gubler on 3/27/15.
+//  Copyright (c) 2015 Sam Gubler. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var friends : [Friend] = []
+
+    @IBOutlet weak var tableView: UITableView!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        self.friends = FriendCommand.getListOfFriends()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+
+    func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return countElements(self.friends)
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCellWithIdentifier("FriendTableViewCell", forIndexPath: indexPath) as FriendTableViewCell
+        
+        cell.configureWithFriend(self.friends[indexPath.row])
```

Comment:
I like the fact that you tell the cell to configure itself instead of cluttering up your the ViewController. Nice!